src/main/java/com/rocketden/main/dto/game/StartGameRequest.java

```diff
@@ -0,0 +1,13 @@
+package com.rocketden.main.dto.game;
+
+import com.rocketden.main.dto.user.UserDto;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class StartGameRequest {
+    private String roomId;
+    private UserDto user;
```

Comment:
Small nit but could you call this initiator, or if not then maybe leave a comment mentioning that it's the person sending the POST request? Mainly to help others remember what it's used for down the line